static/app/utils/profiling/renderers/cursorRenderer.tsx

```diff
@@ -0,0 +1,39 @@
+import {mat3, vec2} from 'gl-matrix';
+
+import {FlamegraphTheme} from '../flamegraph/FlamegraphTheme';
+import {getContext, Rect, resizeCanvasToDisplaySize} from '../gl/utils';
+
+class CursorRenderer {
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
+
+  theme: FlamegraphTheme;
+
+  constructor(canvas: HTMLCanvasElement, theme: FlamegraphTheme) {
+    this.canvas = canvas;
+    this.theme = theme;
+    this.context = getContext(canvas, '2d');
+
+    resizeCanvasToDisplaySize(canvas);
```

Comment:
I noticed that this is being called in the constructor of the `TextRenderer` (different PR) as well. Is there a reason this needs to be called for each renderer? Are they different canvases?

Reply:
@Zylphrex Yeah, we usually render things on few different canvases, e.g the webgl code is going to be its own canvas, text rendering is is its own canvas. In general, we often use an overlay canvas, which helps with perf e.g. clicking on a frame means we just need to draw the outline rect on the overlay canvas instead of redraw the entire canvas.